pkg/extender/extender.go

```diff
@@ -1053,21 +1019,11 @@ func (e *Extender) updateVirtLauncherPodPrioritizeScores(
 							// Current node has the volume replica
 							score = replicaNodeScore
 							nodeHasAttachedVolume := e.nodeHasAttachedVolume(dnode, volInfo)
-
-							if podBeingLiveMigrated != nil {
-								// LiveMigration in progress
-								// If existing pod is not using attached volume and current node has attached volume, assign a higher score to it
-								if !isExistingPodUsingLocallyAttachedVolume && nodeHasAttachedVolume {
-									storklog.PodLog(pod).Infof("Node %v has attached volume", dataNode)
-									score = int64(2 * nodePriorityScore)
-								}
-							} else {
-								if nodeHasAttachedVolume {
-									// There is no LiveMigration in progress
-									// When a new pod gets scheduling request
-									// assign a higher score for local volume attachment
-									score = int64(2 * nodePriorityScore)
-								}
+							if nodeHasAttachedVolume {
+								// There is no LiveMigration in progress
+								// When a new pod gets scheduling request
+								// assign a higher score for local volume attachment
```

Comment:
nit: this comment does not make sense to me. If live migration is NOT in progress, there will not be any volume attachment yet.

Reply:
This is a comment from the previous commit where it made sense based on the particular conditional statement it was inside. Now the corresponding change is gone and it's pointless here. Thanks for pointing. Removing this.